Part1 (Ilia Gusevskii)

IamAfli opened the pull request:
Lambdas homework.
Part1 from 19.06.2017

UPD 21/06/2017
replased Arrays.sort(persons, (p1, p2) -> Integer.compare(p1.getAge(), p2.getAge())); with lambda exp

IamAfli committed:
Feat: sortPersonsByAge() in Lambdas01Exercise released

IamAfli committed:
Feat: findFirstWithAge30() in Lambdas01Exercise released using com.google.common.collect.FluentIterable

IamAfli committed:
Refactor: refactored Lambdas01Exercise methods using anonymous classes

IamAfli committed:
Feat: relesed sortPersonsByAge() and findFirstWithAge30() in Lambdas02Exercise using lambdas

IamAfli committed:
Feat: released generic0() in Lambdas03Exercise

IamAfli committed:
Feat: released generic1() in Lambdas03Exercise

IamAfli committed:
Feat: released generic2() in Lambdas03Exercise

IamAfli committed:
Feat: strSum() in Lambdas03Exercise released

IamAfli committed:
Feat: strSum2() in Lambdas03Exercise released

IamAfli committed:
Refactor: rempalced comparation of integers in sortPersonsByAge() method in Lambdas01Exercise and Lambdas02Exercise to Integer.compare()

senia-psm commented on src/test/java/lambda/part1/exercise/Lambdas02Exercise.java:
```diff
@@ -19,6 +22,7 @@ public void sortPersonsByAge() {
         };
 
         // TODO use Arrays.sort
+        Arrays.sort(persons, (p1, p2) -> Integer.compare(p1.getAge(), p2.getAge()));
```
Use `Comparator.comparingInt`

IamAfli committed:
Refactor: refactored sortPersonsByAge() useing lambdas

replased Arrays.sort(persons, (p1, p2) -> Integer.compare(p1.getAge(), p2.getAge())) with Comparator.comparingInt

IamAfli commented:
Updated files, according to your remarks.
Part1 pull request:
https://github.com/java8-course/lambda/pull/89

Part2 pull request:
https://github.com/java8-course/lambda/pull/100

2017-06-21 13:15 GMT+03:00 Simon Popugaev <notifications@github.com>:

> *@senia-psm* commented on this pull request.
> ------------------------------
>
> In src/test/java/lambda/part1/exercise/Lambdas02Exercise.java
> <https://github.com/java8-course/lambda/pull/89#discussion_r123207681>:
>
> > @@ -19,6 +22,7 @@ public void sortPersonsByAge() {
>          };
>
>          // TODO use Arrays.sort
> +        Arrays.sort(persons, (p1, p2) -> Integer.compare(p1.getAge(), p2.getAge()));
>
> Use Comparator.comparingInt
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/java8-course/lambda/pull/89#pullrequestreview-45386622>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUMr32p6rk_9AVMsOpmQ75NniaYmm5yaks5sGO0tgaJpZM4N_R9V>
> .
>



-- 
С уважением, Гусевский Илья
